README.md

```diff
@@ -39,13 +39,13 @@ Input:
 Output:
 ```ts
 export interface ExampleSchema {
-  firstName: string;
-  lastName: string;
+  "firstName": string;
```

Comment:
> I use a lint rule to prevent unnecessary quoting of object keys.

I see what you mean, but why do you want to lint auto-generated code? 😄 
What if I'd like to see tabs instead of spaces or single quotes instead of double quotes? Then there are controversy and possible ambiguity in the resulting code ))